.github/workflows/build.yaml

```diff
@@ -13,36 +13,24 @@ concurrency:
   group: ${{ github.head_ref }}-build
   cancel-in-progress: true
 jobs:
-  build_toolchain:
-    name: Toolchain
-    if: github.event.pull_request.draft == false
-    uses: ./.github/workflows/build-toolchain.yaml
-    secrets:
-      GOOGLE_HOPRASSOCIATION_PROJECT: ${{ secrets.GOOGLE_HOPRASSOCIATION_PROJECT }}
-      GOOGLE_HOPRASSOCIATION_CREDENTIALS_REGISTRY: ${{ secrets.GOOGLE_HOPRASSOCIATION_CREDENTIALS_REGISTRY }}
-
   build_hopli:
     name: Hopli
     needs:
-      - build_toolchain
+      - build_hoprd
```

Comment:
`hopli` could be parallel with `hoprd`.

Reply:
By having it as a dependency the hopli build can re-use all deps artifact builds and should be way faster.